WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -142,12 +143,12 @@ public static void advertiseImageOptimization(final Context context,
         DialogInterface.OnClickListener onClickListener = new DialogInterface.OnClickListener() {
             @Override
             public void onClick(DialogInterface dialog, int which) {
-                if (which == DialogInterface.BUTTON_POSITIVE) {
-                    if (AppPrefs.isImageOptimize()) {
-                        // null or image optimization already ON. We should not be here though.
-                    } else {
-                        AppPrefs.setImageOptimize(true);
-                    }
+                String propertyValue = (which == DialogInterface.BUTTON_POSITIVE) ? "on" : "off";
+                AnalyticsTracker.track(AnalyticsTracker.Stat.APP_SETTINGS_OPTIMIZE_IMAGES_POPUP_TAPPED,
+                        Collections.singletonMap("option", propertyValue));
+
+                if (which == DialogInterface.BUTTON_NEGATIVE) {
+                    if (AppPrefs.isImageOptimize()) AppPrefs.setImageOptimize(false);
                 }
```

Comment:
Nitpick: These two conditions could be grouped into a single `if` statement.

Reply:
Refactored in https://github.com/wordpress-mobile/WordPress-Android/pull/19581/commits/51ca8afa792ac79662aedf7339ef56280990564a. 🙇‍♀️